recipes/ncvis-climate/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "ncvis-climate" %}
+{% set version = "2022.08.28" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SEATStandards/ncvis/archive/refs/tags/{{ version }}.tar.gz
+  sha256: a522926739b2a05ef0b436fe67a2014557f9e5fecf3b7d7700964e9006a4bf3e
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - wxwidgets
+    - libnetcdf
+    - make
+    - {{ compiler('cxx') }}
+    - {{ cdt('libxxf86vm-devel') }}      # [linux]
+  host:
+    - libnetcdf
+    - wxwidgets
+  run:
+    - libnetcdf
+    - wxwidgets
+
```

Comment:
You can remove these from run. The packages above declare themselves as runtime dependencies